libraries/stream-collator/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Adding strict null checks is a good idea, but outside the scope of this PR (which is supposed to bring over the current code and get it building).